packages/styling/etc/styling.api.md

```diff
@@ -130,7 +130,7 @@ export function getScreenSelector(min: number, max: number): string;
 export function getTheme(depComments?: boolean): ITheme;
 
 // Warning: (ae-internal-missing-underscore) The name getThemedContext should be prefixed with an underscore because the declaration is marked as "@internal"
-//
+// 

```

Comment:
Consider setting your editor to trim unnecessary whitespace to reduce the diff here https://stackoverflow.com/a/30884298/986602